snapcraft/storeapi/v2/_api_schema.py

```diff
@@ -198,7 +198,9 @@
                                 "type": ["string", "null"],
                             },
                         },
-                        "required": ["name", "creation-date", "pattern"],
+                        # pattern is documented as required but is not returned,
```

Comment:
So the API is not passing their own schema?  Is the store team aware?

Reply:
yes, they are fixing it. I suspect (and hope) this is a bug in the documentation and not the schema itself